scripts/irods/database_upgrade.py

```diff
@@ -152,6 +152,13 @@ def run_update(irods_config, cursor):
                         ['mysql', '='.join(['--defaults-file', f.name]), irods_config.database_config['db_name']],
                         stdin=sql_file)
 
+        # Add a new column that will be responsible for holding the fractional part of the modify_ts
+        # for a resource. The value stored will be in milliseconds.
+        if irods_config.catalog_database_type == 'oracle':
+            database_connect.execute_sql_statement(cursor, "alter table R_RESC_MAIN add (modify_ts_millis varchar(3) default '000');")
```

Comment:
we confirm this parentheses notation is correct?   looks a little different than above oracle examples.

Reply:
That is based on their documentation, websites I found, and chatgpt.

But, I can't claim it works since I don't have a way to test it.

Reply:
fair enough.   we'll fix it if we hear from anyone.